front/src/containers/SignIn/index.js

```diff
@@ -1,13 +1,65 @@
 import React, {Component} from 'react'
+import {withRouter} from 'react-router-dom'
+import {toast} from 'react-toastify';
 
-import SignInForm from '../../components/SignInForm'
+import AuthorizationForm from '../../components/SignInForm/authorizationForm'
+import RegistrationForm from '../../components/SignInForm/registrationForm'
+
+import * as userInfo from '../../services/api/userInfoFetch'
+
+import '../../CommonStylesheets/formItems.scss'
+import '../../CommonStylesheets/form.scss'
 
 class SignIn extends Component {
+
+    constructor (props) {
+        super(props);
+        this.onAuthorizationSubmit = this.onAuthorizationSubmit.bind(this);
+        this.onRegistrationSubmit = this.onRegistrationSubmit.bind(this);
+    }
+
+    onAuthorizationSubmit (userData) {
+        userInfo.authorizeUser(userData)
+            .then(response => {
+                if (!response.isExistedUser) {
```

Comment:
Potential security issue. Never expose to intruder additional information. 401 Http Status Code is enough here